README.md

```diff
@@ -223,6 +223,121 @@ localhost. The 'make html' command first downloads the OpenAPI schema for the pl
 in ``docs/_static/api.json``. You should add this file to git. This file will then provide data
 needed to display the restapi.html page in the root of the built docs. 
 
+# Travis configuration
+
+This repository also provides a script for generating a Travis configuration. The script should be
+run with the following command.
+
+   ``$ ./generate_travis_config.py your_plugin_name --pypi-username your_pypi_username``
```

Comment:
The help text shows the `your_plugin_name` as `plugin_name` and it is the last parameter after the kwargs.

Reply:
i updated it to the same order as the help text